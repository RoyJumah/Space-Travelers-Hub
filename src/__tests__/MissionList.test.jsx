import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import MissionsList from '../components/missions/MissionList';
import { Provider } from 'react-redux';
import store from '../redux/ConfigureStore';

describe('Display missionsList component', () => {
  test('it matches snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <MissionsList />
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  })

  test('should return  MissionsLIst Component', () => {
    render(
      <Provider store={store}>
        <MissionsList />
      </Provider>
    );
  })
})
