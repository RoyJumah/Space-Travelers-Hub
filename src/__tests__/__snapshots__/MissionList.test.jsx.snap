// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display missionsList component it matches snapshot 1`] = `
<table
  className="mission-table"
>
  <thead>
    <tr>
      <th>
        Mission
      </th>
      <th>
        Description
      </th>
      <th>
        Status
      </th>
      <th>
        Â 
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
